addons/project/static/src/views/project_task_kanban/project_task_kanban_compiler.js

```diff
@@ -0,0 +1,50 @@
+/** @odoo-module **/
+
+import { KanbanCompiler } from "@web/views/kanban/kanban_compiler";
+import { append, createElement } from "@web/core/utils/xml";
+
+export class ProjectTaskKanbanCompiler extends KanbanCompiler {
+    setup() {
+        super.setup();
```

Comment:
We could also check if the element with`.subtask_list_button` class and `div.kanban_bottom_subtasks_section` are there to be sure we will never have one of both.
```suggestion
        super.setup();
        // ensure both elements are compiled to correctly render the component
        this.subtaskListComponentCompiled = {
            button: false,
            component: false,
        };
```